doc/source/cluster/cluster_under_construction/ray-clusters-on-vms/user-guides/configuring-autoscaling.rst

```diff
@@ -2,55 +2,44 @@
 
 .. _deployment-guide-autoscaler-under-construction:
 
-Autoscaling with Ray
---------------------
+Configuring the Ray Autoscaler
+==============================
 
-Ray is designed to support highly elastic workloads which are most efficient on
-an autoscaling cluster. At a high level, the autoscaler attempts to
-launch/terminate nodes in order to ensure that workloads have sufficient
-resources to run, while minimizing the idle resources.
+This page provides an overview of the parameters for the Ray autoscaler. A Ray cluster runs an autoscaler that can modify
+the number of nodes based on the resources set by task, actor or placement group. The autoscaler only considers logical resources (e.g. those specified
+in ray.remote and displayed in ray status), with the exception of object_store_memory which it does not consider. If a user tries to launch
+an actor, task, or placement group but there are insufficient resources, the request will be queued. The autoscaler runs a bin-packing algorithm
+to determine the list of nodes the cluster should use to satisfy the demand, and adds / removes nodes from the cluster to maximize the
+cluster utilization while providing sufficient resource to execute the requests.
 
-It does this by taking into consideration:
+Parameters
+==========
 
-* User specified hard limits (min/max workers).
-* User specified node types (nodes in a Ray cluster do _not_ have to be
-  homogenous).
-* Information from the Ray core's scheduling layer about the current resource
-  usage/demands of the cluster.
-* Programmatic autoscaling hints.
+**max_workers[default_value=2, min_value=0]**: Indicates the maximum number of workers the cluster will have at any given time. This number
+excludes the head node where there is always exactly one.
```

Comment:
Maybe split it into sentences, something like
`This value excludes the head node -- there is always exactly one head node.`